internal/pkg/template/templates/addons/ddb/env/access_policy.yml

```diff
@@ -11,9 +11,7 @@ Parameters:
 Resources:
   {{logicalIDSafe .Name}}DdbTableAccessPolicy:
     Metadata:
-      'aws:copilot:description': !Sub
-        - 'An IAM managed policy for your service to access the ${Table} DDB table of your environment'
-        - Table: { Fn::ImportValue: { Fn::Sub: "${App}-${Env}-{{logicalIDSafe .Name}}TableName" } }
+      'aws:copilot:description': 'An IAM managed policy for your service to access the DDB table of your environment'
```

Comment:
What if there is more than one DDB table in the env? Does that matter? 🤔 "...access **a** DDB table **in**..."??

Reply:
good call